src/utils/graphql/rsk/operations/getRewardClaimed.graphql

```diff
@@ -0,0 +1,14 @@
+query getRewardClaimed($user: String) {
+  rewardsEarnedHistoryItems(
+    where: { user: $user, action: RewardClaimed, amount_gt: 0 }
```

Comment:
We should discuss this - if all the rewards are going to be in the same table (as they currently are), then all the rewards could be retrieved at once and ordered by timestamp together. At the moment, it doesn't seem to be ordering them by timestamp correctly, I think because it's merging this query and the getRewardWithdrawn query,

But, if they end up being split into separate tables, then this way is best. Let's talk about it with product